### ISSUE TITLE
aria-label is still missing on the enabled Save button in tN projects

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/tc-create-app#546]
v1.0.5-rc.10
Moved here from #420
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/abb01e49-e8c1-49ef-823e-b623d477b45e)
